template/utils/config.py

```diff
@@ -83,8 +84,8 @@ def add_args(cls, parser):
     parser.add_argument(
         "--neuron.epoch_length",
         type=int,
-        help="The default epoch length (how often we set weights).",
-        default=0,
+        help="The default epoch length (how often we set weights, measured in 12 second blocks).",
+        default=5,
```

Comment:
```suggestion
        default=100,
```
should have reasonable default behaviour. If user wants to set weights in a nonstandard way for testing our their use-case, they should be aware of this flag and use it